package.json

```diff
@@ -4,20 +4,20 @@
 	"version": "4.0.1",
 	"description": "Connect your components up to that address bar.",
 	"main": "dist/preact-router.js",
-	"module": "dist/preact-router.module.js",
-	"jsnext:main": "dist/preact-router.module.js",
+	"module": "dist/preact-router.mjs",
+	"jsnext:main": "dist/preact-router.mjs",
```

Comment:
Could we revert this? It might be overly cautious, but there's no (functional) reason to subject these to Node's extension semantics, as Node never reads them. As unlikely as it is to be an issue, there are some older tools that will read `"module"` but will trip over `.mjs`, making this a breaking change.

A post-build `cp` to `.mjs` (if I'm remembering Microbundle's key preference right) would be sufficient.

Reply:
That's a good point! :+1: Another way to fix this without modifying the filenames at all would be to add `"type": "module"`. Maybe this would be even safer? And we would not need to copy the files.

Reply:
`"type": "module"` would necessitate that all CJS entries switch to `.cjs`, which has even less support.

`.mjs` got some popularity and use a few years before `.cjs` & the current Node extension semantics were specc'd and solidified.